src/Components/ComparePublicationData/ComparePublicationDataModal.tsx

```diff
@@ -264,44 +251,8 @@ const ComparePublicationDataModal: FC<ComparePublicationDataModalProps> = ({
       }
       setContributors(tempContributors);
     }
-    !isDuplicate && enrichImportPublicationAuthors().then();
-  }, [importPublication, state.selectedPublication, isDuplicate]);
-
-  useEffect(() => {
-    async function identifyCristinPersonsInContributors_And_CreateListOfIdentified() {
```

Comment:
klarer ikke skjønne om denne koden burde vært brukt

Reply:
Blir identified fra state brukt til noe?

Reply:
nope. kanskje vi har slettet for mye tidligere en gang. men ingen har meldt funksjonaliteten savnet,så da så...